src/main/resources/db/migration/h2/V1.11__skills.sql

```diff
@@ -1,47 +1,38 @@
-CREATE TABLE IF NOT EXISTS `choice`
+CREATE TABLE IF NOT EXISTS `answer`
```

Comment:
Be careful updating existing migrations, id prefer to write new migrations for table renames e.t.c